### PR TITLE
[5.0] Codemirror display tabs

### PR DESCRIPTION
**Pull request by brianteeman**
css change to make the tab character visible in the codemirror editor. 
Especially useful when comparing changed files where the change is from tab to spaces

## To test

either use a prebuilt package or apply pr and `run build:css`

go to the template manager and open a file for editing

paste the text below

```
This is some text with a tab 	in it.
This is some text with two tabs 		in it.
```

### Before
image

### After
image

### CODE REVIEW

**Comment by brianteeman** [comment]
If preferred I can also add visual characters for spaces, line-breaks etc

**Comment by Scrabble96** [comment]
Does this PR allow for the tabs to show up in the 'differences' view?

**Comment by brianteeman** [comment]
no because that doesnt use codemirror

File: build/media_source/plg_editors_codemirror/css/codemirror.css [hunk 1]
```diff
@@ -42,3 +42,14 @@
 .CodeMirror-markergutter-mark:after {
   content: "\25CF";
 }
+
+/* make the tabs a little more visible */
+.cm-tab{
```
Comment (Quy) [inline build/media_source/plg_editors_codemirror/css/codemirror.css]:
```suggestion
.cm-tab {
```

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 5.0-dev. No new features will be merged into Joomla! 4.3 series. Joomla! 4.4 series is a bridge release to make migration from Joomla! 4 to 5 as smooth as possible.

**Comment by HLeithner** [comment]
is this still needed for cm 6, changes are needed at least...

**Comment by brianteeman** [comment]
Probably not as this was something wanted before the new diff engine. Your call as you also expressed a desire for this.

File: build/media_source/plg_editors_codemirror/css/codemirror.css (hunk 1 above)
Comment (Quy) [reply build/media_source/plg_editors_codemirror/css/codemirror.css]: Please add a space.

**Comment by brianteeman** [comment]
Closing this as this code is not relevant for cm6

**Pull request closed without merging**